theme/scripts/cart.ts

```diff
@@ -285,9 +285,10 @@ export default(function() {
       })
     },
 
-    updateProductInfoBar(variantId: string, action: Action) {
+    //TO-DO: This function should check when the add to cart button is clicked and use a setTimeout to change the text to "Added to cart" and then revert back to "Add to cart" after a few seconds.
+    handleAddToCartButton(variantId: string, action: Action) {
       const addToCartButton = document.querySelector(Cart.SELECTORS.addToCartButton);
-
+      
       if (!addToCartButton) return;
       
```

Comment:
Again my comment is for a line that is out of reach...

The fact that you're calling Cart.init(false) at the end of this function is a smell to me - shouldn't it already have been initialized? Maybe separate out the functionality that binds the addToCart forms into a separate method and call that here instead?

If I were to read `Cart.bindAddToCartForms()` at the bottom of this function I'd have a better sense of what's going on.